source/declarations.tex

```diff
@@ -927,7 +927,7 @@
 For a constexpr function or constexpr constructor
 that is neither defaulted nor a template,
 if no argument values exist such that
-an invocation of the function or constructor could be an evaluated subexpression of a core
+an invocation of the function or constructor would be an evaluated subexpression of a core
```

Comment:
I find this very hard to understand with "would" here. I think part of the problem is that the original wording is unclear. What we mean is:

"For a constexpr function or constexpr constructor that is neither defaulted nor a template, if there does not exist a valid core constant expression whose evaluation would evaluate an invocation of the function or constructor, and, for a constructor, there does not exist a valid declaration of a constant-initialized variable whose evaluation would evaluate an invocation of the function or constructor."

That is, this is really "for all possible core constant expressions", not "for all possible argument values".

I think we can simplify and clarify that:

"A constexpr function that is neither defaulted nor a template is ill-formed, no diagnostic required, if it is not possible for an evaluation of an invocation of the function to be performed while evaluating any valid manifestly constant-evaluated expression."

Reply:
fixed